index.js

```diff
@@ -26,10 +28,17 @@ function Physics (mcData, world) {
   const lavaId = blocksByName.lava.id
   const ladderId = blocksByName.ladder.id
   const vineId = blocksByName.vine.id
+  const waterLike = []
```

Comment:
You can use a `new Set()` which has a `.add()` and a `.has()` methods

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set